### ISSUE TITLE
A11y_Dynamics for Microsoft_My Labor Entries_Keyboard: Focus loss is observed "New Time Entry" button.

### ISSUE THREAD

**Issue by Accessibility56** [OfficeDev/office-ui-fabric-react#10365]
[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
https://cs-labor-test.crm.dynamics.com/main.aspx
Server version: 9.1.0000.6228
Client version: 1.3.1948-190626-193951

**Browser Details: ​**
OS: Windows (Version 1903, Build: 18362.239)
Microsoft Edge 44.17763.1.0, Chrome Version 75.0.3770.142 (Official Build) (64-bit)

**Repro Steps: ​**
1)Launch the Url: https://cs-labor-test.crm.dynamics.com/main.aspx and login with appropriate Credentials
2)Tab to all controls and verify focus order after "New Time Entry" button.
 
**Actual Result: ​**
Focus loss is observed "New Time Entry" button.

**Expected Result: ​**
Focus loss should not be there after "New Time Entry" button. For the next tab it should go the next interactive element.

**User Impact: ​**
Keyboard only users may not understand where the focus is if focus loss is observed.

**MAS Reference: ​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}
[Bug_10365_Focus loss issue.pptx](https://github.com/OfficeDev/office-ui-fabric-react/files/3577527/Bug_10365_Focus.loss.issue.pptx)

**Comment by Accessibility56** [comment]
Copied from https://microsoft.visualstudio.com/OSGS/_workitems/edit/23027348

**Comment by KevinTCoughlin** [comment]
@Accessibility56 what version of the `office-ui-fabric-react` library is in use on the page showcasing the above behavior?

**Comment by Accessibility56** [comment]
Office Fabric Version: "office-ui-fabric-react": "^6.9.0",

**Comment by dzearing** [comment]
Can't access the test page. No details in this bug on repro steps. If you are giving no effort in deducing an accessibility bug for your app, we can't help. As this is right now, there is no action we can take.

If you deduce that you are using a Fabric component and that it has a missing behavior causing keyboard accessibility issues, please be clear about the repo. Which component, what were the props passed in, have you repro'd it in isolation?

If you put in that effort and have a clear repro and expected/resulted listed for us, such that we can take action, we're glad to help.

You're also on an incredibly old build of Fabric. If you're falling behind so far, there may likely be fixes that you've missed. Please make sure to test using the latest bits to deduce if you need to just upgrade your dependency.
